keyboards/ergodox_ez/keymaps/mac_win_german_programmer/keymap.c

```diff
@@ -0,0 +1,357 @@
+#include "ergodox_ez.h"
+#include "debug.h"
+#include "action_layer.h"
+#include "version.h"
+
+
+#include "keymap_german.h"
+
+#include "keymap_nordic.h"
+
+
+
+enum custom_keycodes {
+  PLACEHOLDER = SAFE_RANGE, // can always be here
+  EPRM,
```

Comment:
This isn't actually needed anymore. `EEP_RST` does this, and is a core keycode now.

```suggestion
```